src/callbacks/onResize/addPointClick/spaghettiPlot/onResize/addExposure.js

```diff
@@ -0,0 +1,32 @@
+export default function addExposure() {
+    this.svg.select('.se-exposure-group').remove();
+    if (this.edish.exposure.include) {
+        const exposures = this.edish.exposure.data.filter(
+            d => d[this.edish.config.id_col] === this.edish.clicked_id
+        );
+        const g = this.svg.insert('g', '.supergroup').classed('se-exposure-group', true);
+        const rects = g
+            .selectAll('rect.se-exposure')
+            .data(exposures)
+            .enter()
+            .append('rect')
+            .classed('se-exposure', true)
+            .attr({
+                x: d => this.x(+d.EXSTDY),
+                y: 0,
+                width: d =>
+                    d.EXSTDY !== d.EXENDY ? this.x(+d.EXENDY) - this.x(+d.EXSTDY) : 1, // for single day exposures
+                height: this.plot_height,
+                fill: 'black',
```

Comment:
I think that we probably want to go ahead and move this to the bottom margin. Although it might also be nice to have `settings.exposure_location` as "bottom" or "overlay" if you want to get fancy.